css/main.css

```diff
@@ -952,9 +1060,49 @@ nav {
 
   /* FAQ */
   .faq {
-    height: 2000px;
+    color: #F8AC45;
+		height: auto;
+		padding-left: 23px;
+		padding-right: 23px;
+    padding-top: 200px;
+		background-image: none;
+		text-align: center;
   }
 
+	.faq-machine-dropper-right {
+		display: none;
+	}
+
+	.faq-machine-dropper-left {
+		display: none;
+	}
+
+	#faq-title {
+        font-size: 115px;
+        font-weight: 600;
+        padding-top: 0;
+        text-align: center;
+	}
+
+	.faq-text {
+    font-size: 57px;
+    font-weight: 100;
+    line-height: 76px;
+    letter-spacing: 1px;
+		padding-top: 40px;
+		padding-bottom: 40px;
+	}
+
+	.faq-bold-text {
+		font-family: 'Avenir-Black';
+	}
+
+	.faq-link {
+		color: #FFAD0B;
+		text-decoration: none;
```

Comment:
I would recommend leaving the text-decoration.  Without it, the link is not very discoverable.